docs/setup/settings.asciidoc

```diff
@@ -82,52 +83,59 @@ Elasticsearch nodes on startup.
 `elasticsearch.sniffOnConnectionFault:`:: *Default: false* Update the list of
 Elasticsearch nodes immediately following a connection fault.
 
-`elasticsearch.ssl.alwaysPresentCertificate:`:: *Default: false* Controls whether to always present the certificate specified by
-`elasticsearch.ssl.certificate` or `elasticsearch.ssl.keystore.path` when requested. This setting applies to all requests to Elasticsearch,
-including requests that are proxied for end users. Setting this to `true` when Elasticsearch is using certificates to authenticate users can
-lead to proxied requests for end users being executed as the identity tied to the configured certificate.
+`elasticsearch.ssl.alwaysPresentCertificate:`:: *Default: false* Controls {kib}'s behavior in regard to presenting a client certificate when
+requested by {es}. This setting applies to all outbound SSL/TLS connections to {es}, including requests that are proxied for end users.
++
+WARNING: Setting this to `true` when {es} is using certificates to authenticate end users with a PKI realm can lead to proxied requests for
+end users being executed as the identity tied to the {kib} server.
```

Comment:
Continued attempt at simplification:
```suggestion
are executed as the identity that is tied to the {kib} server.
```

Reply:
This isn't entirely accurate. If this setting is enabled, end users _may_ be mistaken for the Kibana server. I'll apply this suggestion, but change `are` to `may be`.